yellowstone-grpc-geyser/src/plugin.rs

```diff
@@ -174,6 +174,13 @@ impl GeyserPlugin for Plugin {
         status: SlotStatus,
     ) -> PluginResult<()> {
         let inner = self.inner.as_ref().expect("initialized");
+        // if plugin run on genesis of network, notify_end_of_startup will not be triggered, and slot 0 will not
+        // come through this function.
+        if parent.eq(&Some(0)) {
```

Comment:
does `.eq` have any benefits instead of `==`?
```suggestion
        if parent == Some(0) {
```

Reply:
Sometimes compiler complains, but this one doesn't seem to matter so I'll update